virt/virt-4-12-release-notes.adoc

```diff
@@ -47,8 +47,10 @@ The SVVP Certification applies to:
 //CNV-18488 SVVP for 4.12
 
 //CNV-21611-2
-{VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
+* {VirtProductName} no longer uses the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo. {VirtProductName} is now represented by the image:virt-icon.svg[{VirtProductName},40,40] logo for versions 4.9 and later.
```

Comment:
no longer uses the image -> no longer uses the old image 
is now represented by the image -> is now represented by the new image

Reply:
Out of scope of this PR.